Scripts/extract_release_notes.rb

```diff
@@ -21,7 +21,7 @@
 
 def replace_pr_number_with_markdown_link(string)
   string.gsub(/\#\d*$/) do |pr_number|
-    "[#{GITHUB_URL}/pull/#{pr_number.gsub('#', '')}](#{pr_number})"
+    "[#{pr_number}](#{GITHUB_URL}/pull/#{pr_number.gsub('#', '')})"
```

Comment:
This resulted in a link named with the full PR URL pointing to a non-existing internal anchor in the page rendering it 🤦